src/main/java/com/github/jchartier/slackbot/pomodoro/PomodoroCommandManager.java

```diff
@@ -76,6 +94,32 @@ private void stopPomodoro(SlackUser slackUser) {
         pomodoroNotificationService.stopPomodoro(slackUser);
     }
 
+    private void listActivePomodoro(SlackUser slackUser) {
+
+        List<SlackUser> activeUsers = listActivePomodoros();
+
+        if (activeUsers.isEmpty()) {
+
+            slackSession.sendMessageToUser(slackUser, "No active pomodoro found", null);
+        } else {
+
+            String message = listActivePomodoros().stream()
+                    .map(SlackPersona::getRealName)
+                    .reduce("", (result, name) -> (result + "- " + name + "\n"));
```

Comment:
I would recommend using the `•` char. Plus it would be a good idea to add it in a constant because we seem to be using it quite a lot.